token/sdk/tms/db/tms.go

```diff
@@ -157,3 +191,20 @@ func (p *PostInitializer) ConnectNetwork(networkID, channel, namespace string) e
 	}
 	return nil
 }
+
+type TransactionFilter struct {
+	ttxDB   *ttxdb.DB
+	auditDB *auditdb.DB
+}
+
+func (t *TransactionFilter) Accept(txID string, env []byte) (bool, error) {
+	status, _, err := t.ttxDB.GetStatus(txID)
```

Comment:
to avoid some nesting:
```
if status, _, err := t.ttxDB.GetStatus(txID); err == nil && status != ttxdb.Unknown {
 return true, nil
}
if status, _, err := t.auditDB.GetStatus(txID); err != nil || status == ttxdb.Unknown {
 return status != ttxdb.Unknown, err
}
return true, nil
```